### PR TITLE
Bitcoin Core 0.21.2

### PR DESCRIPTION
**Pull request by fanquake**
Adds a release announcement for 0.21.2. Modeled after #792.

### CODE REVIEW

**Comment by laanwj** [comment]
Thanks! I think you got everything right, including the magnet link.

ACK 850d30d5f68207993d060da3bc5e485825fd9e82

File: _releases/0.21.2.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+- MarcoFalke
+- Pavol Rusnak
+- Pieter Wuille
+- prayank23
+- Rafael Sadowski
+- W. J. van der Laan
+
+
+As well as to everyone that helped with translations on
+[Transifex](https://www.transifex.com/bitcoin/bitcoin/).
```
Comment (harding) [inline _releases/0.21.2.md:136]: Needs an additional line with `{% endgithubify %}` Example: https://github.com/bitcoin-core/bitcoincore.org/pull/792/files#diff-0bbaa9abb1d385dad9ae6fe573deb1f53269e1ef5daf9f024e95b2f7048fb2ccR1191

File: _releases/0.21.2.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+- #20182 Build with --enable-werror by default, and document exceptions (hebasto)
+- #20535 Fix intermittent feature_taproot issue (MarcoFalke)
+- #21663 Fix macOS brew install command (hebasto)
+- #22279 add missing ECCVerifyHandle to base_encode_decode (apoelstra)
+- #22730 Run fuzzer task for the master branch only (hebasto)
+
+### GUI
+
+- #277 Do not use QClipboard::Selection on Windows and macOS. (hebasto)
```
Comment (harding) [inline _releases/0.21.2.md:105]: The GUI PRs need to be prefixed with `gui`, e.g.

```text
- gui#277 Do not use QClipboard::Selection on Windows and macOS. (hebasto)
```

That way the automatic PR links will point to the correct repo.

**Review by harding — CHANGES_REQUESTED** [review]

**Comment by harding** [comment]
Tested ACK a2820805aed231e1aec39aad7239d17a050c81d4 .  Checked local preview and locally ran the CI tests.  Thanks!

**Review by harding — APPROVED** [review]

**Pull request merged**